Assets/Scenes/Main.unity

```diff
@@ -1957,6 +2019,17 @@ PrefabInstance:
       propertyPath: m_IsActive
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 7544585871087336863, guid: 5fbb7c7c8ec47e849afba1b658f3e07e,
```

Comment:
Ik meen mij te herinneren dat dit testdata was? Of heet deze alleen 'testDecal'?